examples/index.js

```diff
@@ -100,6 +108,63 @@ class CustomArrows extends Component {
   }
 }
 
+class ControlPlay extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { playing: false };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ playing: true });
+    }, 5000);
+  }
+
+  render() {
+    return (
+      <Slider infinite swipe draggable autoPlay={this.state.playing} >
+        <button>Prev</button>
+        <Slides {...this.props}>
+          <div style={slideStyle}><h1>1</h1></div>
+          <div style={slideStyle}><h1>2</h1></div>
+          <div style={slideStyle}><h1>3</h1></div>
+          <div style={slideStyle}><h1>4</h1></div>
+          <div style={slideStyle}><h1>5</h1></div>
+          <div style={slideStyle}><h1>6</h1></div>
+        </Slides>
+        <button>Next</button>
+      </Slider>
+    );
+  }
+}
+
+class SetSlide extends Component {
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.refs.slider.setState({ currentSlide: 5 });
+    }, 5000);
```

Comment:
SetSlide should have the state here. Try to think about it. I will write a detailed comment little later.